internal/docker/supported.go

```diff
@@ -81,5 +91,26 @@ func cleanupDockerResource(pool *dockertest.Pool, resource *dockertest.Resource)
 	if strings.Contains(err.Error(), "No such container") {
 		return nil
 	}
-	return fmt.Errorf("Failed to cleanup local container: %s", err)
+	return fmt.Errorf("failed to cleanup local container: %s", err)
+}
+
+// splitImage takes the WithDatabaseImage option and separates
+// it into repo + tag. If a tag is not found, it sets the tag to latest
+func splitImage(opt ...Option) (string, string, error) {
+	opts := GetOpts(opt...)
+
+	separated := strings.Split(opts.withContainerImage, ":")
+	separatedlen := len(separated)
+
+	switch separatedlen {
+	case 1:
+		return separated[0], "11", nil
```

Comment:
The comment mentioned this sets tag to latest, however it seems you set it to `11` here.  I believe this is the version we support for postgres right now but it is not the latest. We should also perhaps error if `separated[0] != "postgres"`